src/components/Form.js

```diff
@@ -171,3 +237,39 @@ const SubmitMessage = styled.h3`
   color: green;
   text-align: center;
 `;
+
+const CheckboxWrapper = styled.div`
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: start;
```

Comment:
`start` is the default value of `justify-content`. Therefore, you don't need to declare this property.